### ISSUE TITLE
scream_default_output.yaml frequency params should not depend on CIME HIST_* vars

### ISSUE THREAD

**Issue by ambrad** [E3SM-Project/scream#1824]

**Comment by PeterCaldwell** [comment]
In previous E3SM releases, HIST_* controlled **coupler** output frequency. **Atmosphere** output frequency was controlled by nhtfrq in user_nl_eam. It was a mistake to ever connect our atm output with HIST_* if for no other reason than HIST_* is just the wrong variable name. 

A bigger question is whether we should have an xmlchange variable or atmchange variable which controls atm output frequency. It is annoying to have to manually change scream_output.yaml to change output frequency. I assume the current HIST_* coupling was born from the desire to make it easy to change this variable that we are always tinkering with. One question worth answering is whether we *need* to modify this variable in scriptable format for some of our tests to work. If that's not the case, I vote for forcing people to modify scream_output.yaml to change output for v1, leaving xmlization of output as a v1.1 task.

**Comment by ambrad** [comment]
v1.1 is fine. I was just noting this after working to make the default output support the pg2 grid in tests. Note variable replacement in this yaml file will support non-CIME variables after the pg2 PR goes in, so we could introduce new ones like DEFAULT_OUTPUT_FREQUENCY, etc.

**Comment by PeterCaldwell** [comment]
> v1.1 is fine.

To be clear, I was agreeing with you that we should make your suggested change (decoupling output.yaml freq from CIME HIST_*) ASAP. The thing I was advocating as a v1.1 task was an overhaul or additional xml layer over the top of the output.yaml implementation we have. 

> Note variable replacement in this yaml file will support non-CIME variables after the pg2 PR goes in, so we could introduce new ones like DEFAULT_OUTPUT_FREQUENCY, etc.

I'm not sure I understand this comment. I think you're just saying that your pg2 PR decouples output.yaml from the variable names CIME expects? If so, I like it.

**Comment by ambrad** [comment]
> I'm not sure I understand this comment. I think you're just saying that your pg2 PR decouples output.yaml from the variable names CIME expects? If so, I like it.

Right. To avoid making multiple default output yaml files, one for each physics grid, I modified the infrastructure to do variable replacement in scream_default_output.yaml using variables other than just CIME ones. There's a new one called PHYSICS_GRID_TYPE. We could introduce others.

**Comment by bartgol** [comment]
I like that. I was chatting with Jim at some point, to allow some ATM variables, to be used in xml/yaml file config. Looking forward to see the infrastructure upgrades!

**Comment by ambrad** [comment]
Closing this as it's not important. Testing and doing production runs are doing fine.
